awswrangler/athena/_read.py

```diff
@@ -469,6 +467,7 @@ def _resolve_query_without_cache(
                 pyarrow_additional_kwargs=pyarrow_additional_kwargs,
             )
         finally:
+            _logger.debug("Deleting catalog table: %s", name)
```

Comment:
Hmm, I would either remove this or add the log call inside the catalog method directly. Saves us LOC all over the codebase

Reply:
Agreed, added